src/transform/transform.ts

```diff
@@ -1057,3 +1088,14 @@ function getEnumForDiscriminatorValue(discValue: string, enums: Array<ChoiceValu
   }
   throw console.error(`failed to find discriminator enum value for ${discValue}`);
 }
+
+// convert comments that are in Markdown to html and then to plain text
+export function parseComments(comment: string): string {
```

Comment:
Does this actually need to be exported?  Looks like it's used locally.

Reply:
Nope, I removed the export.